app/src/main/java/com/mobile/tandil/javabaseproject/mvp/presenter/ParkingPresenter.java

```diff
@@ -6,9 +6,16 @@ public class ParkingPresenter implements ParkingContract.Presenter {
 
     private final ParkingContract.Model parkingModel;
     private final ParkingContract.View parkingView;
+    private final int testValue = 99;
 
     public ParkingPresenter(ParkingContract.Model parkingModel, ParkingContract.View parkingView) {
         this.parkingModel = parkingModel;
         this.parkingView = parkingView;
     }
+
+    @Override
+    public void showAvailableParkingSpots(){
+        parkingModel.setAvailableParkingSpots(testValue);
+        parkingView.displayAvailableParkingSpots(String.valueOf(parkingModel.getAvailableParkingSpots()));
```

Comment:
It's not the presenter's responsibility to parse the numbers, it's the model's responsibility

Reply:
done